### PR TITLE
feat: stored iterations reduced to one

### PR DESCRIPTION
**Pull request by VGeorgee**

### CODE REVIEW

**Comment by battila7** [comment]
@VGeorgee Let me know once this is ready for review 🔥

File: sokutei.h [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+        SOKUTEI_MAX_MEASURED_ITERATIONS;
+
+int sokutei_current_iteration = 0;
+
+
+void sokutei_iteration_finish_handler(){
+    int index;
+    for(index = 0; index < (SOKUTEI_MAX_COUNTER_COUNT * MAX_SIZE_OF_TYPES); index++){
+        sokutei_counters[index] = 0;
+    }
```
Comment (battila7) [inline sokutei.h:152]: I think you can use [`memset`](https://en.cppreference.com/w/c/string/byte/memset) here.

**Review by battila7 — CHANGES_REQUESTED** [review]
One technical change requested, otherwise it looks fine 👍

File: sokutei.h (hunk 1 above)
Comment (VGeorgee) [reply sokutei.h:152]: Thank you for the review @battila7! As far as I know, memset is a part of string.h header, which we decided not to include, for dependency reasons. Are you sure I should use it?

Comment (battila7) [reply sokutei.h:152]: Fuck this shit, you're right. I was a bit out-of-scope when reviewing 😄

**Review by battila7 — APPROVED** [review]

**Comment by battila7** [comment]
@VGeorgee Feel free to merge when you feel like it.

**Pull request merged**